activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -102,26 +102,28 @@ def initialize(*addresses)
 
       # Increment a cached value. This method uses the memcached incr atomic
       # operator and can only be used on values written with the :raw option.
-      # Calling it on a value not stored with :raw will initialize that value
-      # to zero.
+      # Calling it on a value not stored with :raw will fail and return nil.
+      # With the :initial option, its vlaue is used for a new counter.
+      # Otherwise, an operation to a nonexistant value will fail and return nil.
```

Comment:
```suggestion
      # Calling it on a value not stored with :raw will fail and return nil.
      # With the :initial option, its value is used for a new counter.
      # Otherwise, an operation to a nonexistent value will fail and return nil.
```

Reply:
Applied, thanks